netlogo-gui/src/main/app/infotab/InfoFormatter.scala

```diff
@@ -44,12 +44,12 @@ object InfoFormatter {
                          MaxParsingTimeMillis)
       .markdownToHtml(str)
 
-  def wrapHtml(body: HTML, fontSize: Int = defaultFontSize): HTML = {
-    "<html><head>"+styleSheet(fontSize)+"</head><body>"+body+"</body></html>"
-  }
+  def wrapHtml(body: HTML, fontSize: Int = defaultFontSize): HTML =
+    "<html><head>"+styleSheet(fontSize)+"</head><body>"+
+      body.replaceAll("<li>", "<li>&bull; ")+
```

Comment:
Oh, you're totally right! I missed that you were just replacing `<li>` with `<li>&bull;`, totally my mistake! This looks good. I plan to merge it in shortly.